make/appimage.mk

```diff
@@ -3,13 +3,17 @@ APPIMAGE_DIR = $(PLATFORM_DIR)/appimage
 APPIMAGETOOL = appimagetool-x86_64.AppImage
 APPIMAGETOOL_URL = https://github.com/AppImage/AppImageKit/releases/download/13/appimagetool-x86_64.AppImage
 
+RELEASE_DATE := $(shell date --iso-8601)
```

Comment:
Shouldn't this use the commit date?

Reply:
A tag is basically just saying some commit is special.

> Do you build the packages just after tagging the release?

Normally yes but it seems to me that the build date is less informative.

Reply:
I agree with @Frenzie: build date is useless, better use the commit date:
```diff
--- i/Makefile
+++ w/Makefile
@@ -3,11 +3,12 @@ KOR_BASE?=base
 
 include $(KOR_BASE)/Makefile.defs
 
+RELEASE_DATE := $(shell git show -s --format=format:"%cd" --date=short HEAD)
 # We want VERSION to carry the version of the KOReader main repo, not that of koreader-base
-VERSION:=$(shell git describe HEAD)
+VERSION := $(shell git describe HEAD)
 # Only append date if we're not on a whole version, like v2018.11
 ifneq (,$(findstring -,$(VERSION)))
-	VERSION:=$(VERSION)_$(shell git show -s --format=format:"%cd" --date=short HEAD)
+  VERSION := $(VERSION)_$(RELEASE_DATE)
 endif
 
 # releases do not contain tests and misc data
--- i/make/appimage.mk
+++ w/make/appimage.mk
@@ -3,8 +3,6 @@ APPIMAGE_DIR = $(PLATFORM_DIR)/appimage
 APPIMAGETOOL = appimagetool-x86_64.AppImage
 APPIMAGETOOL_URL = https://github.com/AppImage/AppImageKit/releases/download/13/appimagetool-x86_64.AppImage
 
-RELEASE_DATE := $(shell date --iso-8601)
-
 update: all
 	# remove old package if any
 	rm -f koreader-appimage-$(MACHINE)-$(VERSION).appimage
--- i/make/linux.mk
+++ w/make/linux.mk
@@ -1,8 +1,6 @@
 LINUX_DIR = $(PLATFORM_DIR)/linux
 LINUX_PACKAGE:=koreader-linux-$(LINUX_ARCH_NAME)$(KODEDUG_SUFFIX)-$(VERSION).tar.xz
 
-RELEASE_DATE := $(shell date --iso-8601)
-
 update: all
 	mkdir -pv \
 		$(INSTALL_DIR)/linux/bin \
```